### PR TITLE
feat(css-main): sticky headers

### PR DESCRIPTION
**Pull request by hinell**
> _**Warning**: Navigation via TOC gets broken._

Add sticky headers:

https://user-images.githubusercontent.com/8136158/204105238-e8210eac-dfdd-405c-bb4f-f75bd935d460.mp4

### CODE REVIEW

**Comment by hinell** [comment]
I can add a `*.js` script that can handle menu navigation via TOC.

**Comment by clason** [comment]
That would be preferable, yes. TOC breaking is a deal-breaker here.

File: css/main.css [hunk 1]
```diff
@@ -96,27 +100,38 @@ h1,
...
+
+h2, h4 {
+  color: var(--accent-color);
 }
 
-h4 {
-  font-size: 1em;
+div.old-help-para {
+    overflow: unset;
+    display: inline;
```
Comment (justinmk) [inline css/main.css:134]: nit: use same indent as elsewhere (2 space)

File: css/main.css [hunk 2]
```diff
@@ -96,27 +100,38 @@ h1,
...
-  font-size: 1.6em;
 }
 
-h2 {
-  font-size: 1.3em;
+/* STICKY HEADERS START */
+
+h1,
+h2.help-heading,
+h3.help-heading
```
Comment (justinmk) [inline css/main.css:115]: should help-specific stuff live in [help.css generated by gen_help_html.lua](https://github.com/neovim/neovim/blob/266a1c61b97313f2a516140fc691dd8be8eed47a/scripts/gen_help_html.lua#L841)? else things get hard to track down.

Comment (hinell) [reply css/main.css:115]: Well, if this one is generated automatically then - yes.

**Comment by hinell** [comment]
Have to be implemented in doc-generating code.

Note: sticky-headers are wreacking navigation

**Pull request closed without merging**